src/auth/session/storage/__tests__/mongodb.test.ts

```diff
@@ -0,0 +1,54 @@
+import * as child_process from 'child_process';
+import {promisify} from 'util';
+
+import * as mongodb from 'mongodb';
+
+import {MongoDBSessionStorage} from '../mongodb';
+
+import {batteryOfTests} from './battery-of-tests';
+import {poll} from './utils';
+
+const exec = promisify(child_process.exec);
+
+const dbURL = new URL('mongodb://shopify:passify@localhost');
+const dbName = 'shopitest';
+
+// SORRY NOT SORRY
+jest.setTimeout(20000);
```

Comment:
was this because of the Docker out of memory (?) error you mentioned?

Reply:
... I should really add this explanation to the comment, shouldn’t I?